doc/source/placement-group.rst

```diff
@@ -0,0 +1,140 @@
+Placement Group
+===============
+
+Ray is great at scheduling tasks and actors based on its resource requirements.
+requirements, it is also important to specify which "place" they should be scheduled. Ray's placement group APIs allow you to place your actors or tasks according to your strategies.
+
+Let's see some scenarios it is useful.
+
+- **Gang scheduling**: Let's say you'd like to place exact number of actors on each GPU node to maximize bandwidth.
+- **Maximizing data locality**: To avoid Ray objects transfer cost, you'd like to pack all actors or tasks as much as possible.
+- **Improving availability**: To improve availability, you'd like to place your actors or tasks into different physical machines as much as possible.
+- **Resource reservation in multi-tenant cluster**: You can reserve resources for your job ahead of time, so that you won't need to worry that resource competition slows down your job.
+
+.. note::
+
+  Ray's placement group is still under active development. Please try out and give us feedback and report bugs!
+  APIs are subject to change in the future.
+
+Quick Start
```

Comment:
IMO you should put this last (as an example). You should start off with *key concepts* first!

Reply:
+1, it would be great to explain what bundles are earlier on!

Reply:
fixed!